chore: change engine version (#196)
chainflip-io/chainflip-docs: pages/mainnet/validator-setup/managing-runtime-upgrades.mdx

<Callout type="info">This process is only required for Major or Minor releases (i.e. 1.x -> 1.y)</Callout>

> We will now go through some examples of how to manage this process on VPS and Kubernetes. Let's imagine an upgrade from `1.X.0` to `1.Y.0`, where `y = x + 1`.

Author: ```suggestion
We will now go through some examples of how to manage this process on VPS and Kubernetes. Let's imagine an upgrade from `1.X.0` to `1.Y.0`, where `Y = X + 1`.
```

```shell
sudo apt update
sudo apt --only-upgrade install chainflip-node
sudo apt install chainflip-engine1.Y
```

You should see the following logs from the new engine:
```shell
journalctl -f -u chainflip-engine1.Y
{"timestamp":"2023-12-11T16:26:38.381549Z","level":"INFO","fields":{"message":"This version '1.Y.0' is incompatible with the current release '1.X.0' at block: 0x4d66…fb38. WAITING for a compatible release version."},"target":"chainflip_engine::state_chain_observer::client"}
```

### After the Runtime Upgrade

Check the logs of the new version of the `chainflip-engine1.Y`. You should see that it is now processing blocks.

> You can now safely disable the old version of the `chainflip-engine`.

Author: ```suggestion
You can now safely disable and remove `chainflip-engine1.X`.
```
